Bejebeje.DataAccess/ReadMe.md

```diff
@@ -4,7 +4,7 @@ In Bash, you run: export ASPNETCORE_ENVIRONMENT=Development
 
 # Create New Migration
 
-dotnet ef migrations --project Bejebeje.DataAccess --startup-project Bejebeje.Api add NewMigrationName
+dotnet ef migrations --project Bejebeje.DataAccess --startup-project Bejebeje.Api add InitialCreate
```

Comment:
is this really required? If i remember correctly there was a .Migrate() or .EnsureMigration() that can be called on the dbcontext.Database to automatically apply pending Migrations